examples/jsm/nodes/core/Node.js

```diff
@@ -100,6 +102,42 @@ Node.prototype = {
 
 	},
 
+	getHash: function() {
+
+		var hash = '{';
+		var prop, obj;
+
+		for(prop in this) {
```

Comment:
@Mugen said

>I can clean things up after merging. Don't be worry about this 

Please be consistent. If users are not expected to follow the three.js code style, then please remove the instructions to do so found in https://github.com/mrdoob/three.js/wiki/How-to-contribute-to-three.js.

>Format whitespace consistently with the rest of code base. See the [an] article Mr.doob's-Code-Style™, and the online code beautifier. It is highly recommended to use ESLint plugin and appropriate settings in your code editor.

/ping @mrdoob

Reply:
@WestLangley I see no problem in fix this... so far I thought that you is referring to the json output.

Reply:
Sorry, just wanted to get this merged quickly :nerd_face:.